xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1150,6 +1239,12 @@ XLA_TEST_F(FlashAttentionBMMScalePaddingMaskSoftmaxBMM,
            Flash_Attention_Training_BMM1_PaddingMask_Softmax_BMM2_BF16) {
   TestImpl_Flash_Attention_Training_BMM1_PaddingMask_Softmax_BMM2<bfloat16>();
 }
+
+// BMM1 - Scale - Softmax - BMM2 fp8
+XLA_TEST_F(FlashAttentionBMMScalePaddingMaskSoftmaxBMMF8,
+           Flash_Attention_Inference_BMM1_NoMask_Softmax_BMM2_F8) {
```

Comment:
It seems this test is failing internally with

UNIMPLEMENTED: No registered implementation for custom call to __cudnn$fmhaSoftmax$f8 for platform CUDA

Is there anything else required than cudnn 9.0 or above?